src/main/java/it/gov/pagopa/ranking/controller/RankingControllerImpl.java

```diff
@@ -16,6 +16,11 @@ public RankingControllerImpl(OnboardingRankingBuildFileMediatorService onboardin
         this.onboardingRankingBuildFileMediatorService = onboardingRankingBuildFileMediatorService;
     }
 
+    @Override
+    public void forceRankingInitiativeEnd(String initiativeId) {
+        onboardingRankingBuildFileMediatorService.forceRankingInitiativeEnd(initiativeId);
```

Comment:
the behavior is not related to the file building, so use a separate service (maybe it.gov.pagopa.ranking.service.initiative.InitiativeConfigService)